torch/tensor.py

```diff
@@ -734,6 +734,11 @@ def _update_names(self, names, inplace):
         else:
             return super(Tensor, self).rename(names)
 
+    def _validate_after_load(self):
+        if not self.is_sparse:
```

Comment:
nit: The logic here seems inverted. *IF* it is sparse, then I should validate sparse coo tensor args (not the other way around!)